setup.py

```diff
@@ -5,26 +5,25 @@
 extras_require = {
     "test": [  # `test` GitHub Action jobs uses this
         "ape-fantom",  # For testing Fantom integration
-        "ape-arbitrum",  # For testing Arbitrum integration
         "ape-optimism",  # Needed for Optimism integration
         "ape-infura",  # Needed for live network tests
-        "pytest>=6.0,<7.0",  # Core testing package
+        "pytest>=6.0",  # Core testing package
         "pytest-xdist",  # multi-process runner
         "pytest-cov",  # Coverage analyzer plugin
         "hypothesis>=6.2.0,<7.0",  # Strategy-based fuzzer
         "pytest-mock",  # Test mocker
     ],
     "lint": [
-        "black>=22.3.0,<23.0",  # auto-formatter and linter
-        "mypy>=0.961,<1.0",  # Static type analyzer
-        "types-requests>=2.27.15",  # NOTE: Needed due to mypy typeshed
+        "black>=22.6.0,<23.0",  # auto-formatter and linter
```

Comment:
```suggestion
        "black>=22.6.0",  # auto-formatter and linter
```

this will cause issues during install!